### ISSUE TITLE
Training with custom dataset

### ISSUE THREAD

**Issue by vivekam101** [kamalkraj/BERT-SQuAD#3]
Hi Kamal,
Can you please share how to do finetuning with custom dataset

**Comment by kamalkraj** [comment]
The pre-trained model comes from https://github.com/huggingface/transformers. 
You can refer to there documentation for fine-tuning on a custom dataset. 
After finetuning you can use this repo for inference. Point the code to new model

**Comment by gurvesh** [comment]
Thanks for for providing this Kamal!

For anyone else interested, I was able to get better performance with the following model which is provided by the transformer library directly (need to edit bert.py):

```python
tokenizer = BertTokenizer.from_pretrained('bert-large-uncased-whole-word-masking-finetuned-squad')
model = BertForQuestionAnswering.from_pretrained('bert-large-uncased-whole-word-masking-finetuned-squad', config=config)
```

**Comment by kamalkraj** [comment]
Hi @gurvesh,
I think the new model is recently released by transformers. Thanks for posting here.
I will add this information to README

What are EM and F1 Score of `bert-large-uncased-whole-word-masking-finetuned-squad` ?

**Comment by gurvesh** [comment]
I think the scores are about the same. But in actual usage, I found the answers given by the finetuned-squad model on random articles I put through the models were much better. YMMV
